### PR TITLE
LLM: add example of aquila

### PR DESCRIPTION
**Pull request by JinBridger**
## Description

Add Aquila example in `python/llm/example/transformers/transformers_int4` using `AquilaChat-7B`

Including `generate.py` and `README.md`.

### CODE REVIEW

File: python/llm/example/transformers/transformers_int4/aquila/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# AquilaChat
```
Comment (jason-dai) [inline python/llm/example/transformers/transformers_int4/aquila/README.md:1]: Just Aquila

File: python/llm/example/transformers/transformers_int4/aquila/README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+In this directory, you will find examples on how you could apply BigDL-LLM INT4 optimizations on AquilaChat models. For illustration purposes, we utilize the [BAAI/AquilaChat-7B](https://huggingface.co/BAAI/AquilaChat-7B) as a reference AquilaChat model.
```
Comment (jason-dai) [inline python/llm/example/transformers/transformers_int4/aquila/README.md:3]: Aquila models

File: python/llm/example/transformers/transformers_int4/aquila/generate.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+import torch
+import time
+import argparse
+
+from bigdl.llm.transformers import AutoModelForCausalLM
+from transformers import AutoTokenizer
+
+# you could tune the prompt based on your own model,
+# here the prompt tuning refers to https://huggingface.co/BAAI/AquilaChat-7B/blob/13577616fd4ff0d21c5735a88d350a68dae120e5/cyg_conversation.py
+AQUILA_PROMPT_FORMAT = "A chat between a curious human and an artificial intelligence assistant. The assistant gives helpful, detailed, and polite answers to the human's questions.###Human: {prompt}###Assistant:"
```
Comment (jason-dai) [inline python/llm/example/transformers/transformers_int4/aquila/generate.py:26]: the prompt is too long

File: python/llm/example/transformers/transformers_int4/aquila/README.md (hunk 1 above)
Comment (JinBridger) [reply python/llm/example/transformers/transformers_int4/aquila/README.md]: Already done as you said.

File: python/llm/example/transformers/transformers_int4/aquila/README.md (hunk 2 above)
Comment (JinBridger) [reply python/llm/example/transformers/transformers_int4/aquila/README.md]: Already done as you said.

File: python/llm/example/transformers/transformers_int4/aquila/generate.py (hunk 3 above)
Comment (JinBridger) [reply python/llm/example/transformers/transformers_int4/aquila/generate.py]: Removed the sentence before `Human:`.

**Review by shane-huang — APPROVED** [review]
LGTM

**Pull request merged**